siriuspy/siriuspy/magnet/normalizer.py

```diff
@@ -31,8 +29,10 @@
 class _MagnetNormalizer:
     """Base class for converting magnet properties: current and strength."""
 
-    def __init__(self, maname, magnet_conv_sign=-1):
+    def __init__(
+            self, maname, magnet_conv_sign=-1, default_strengths_dipole=None):
         """Class constructor."""
+        self.default_strengths_dipole = default_strengths_dipole  # [GeV]
```

Comment:
@fernandohds564 , here comes again the question on class API. before we change to a new python version and resume discussions on our standard coding I suggest we try to keep consistency at least within each class separately. In this case the class uses hidden attributes and associated property and setter decorated methods.

Reply:
Ok. I don't like the idea, but it's done!